WeScan/Scan/ScannerViewController.swift

```diff
@@ -73,25 +73,40 @@ final class ScannerViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        title = NSLocalizedString("wescan.scanning.title", tableName: nil, bundle: Bundle(for: ScannerViewController.self), value: "Scanning", comment: "The title of the ScannerViewController")
+        title = nil
         
         setupViews()
-        setupToolbar()
+        setupNavigationBar()
         setupConstraints()
         
         captureSessionManager = CaptureSessionManager(videoPreviewLayer: videoPreviewLayer)
         captureSessionManager?.delegate = self
         
+        originalBarStyle = navigationController?.navigationBar.barStyle
+        
         NotificationCenter.default.addObserver(self, selector: #selector(subjectAreaDidChange), name: NSNotification.Name.AVCaptureDeviceSubjectAreaDidChange, object: nil)
     }
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
+        setNeedsStatusBarAppearanceUpdate()
+        
         CaptureSession.current.isEditing = false
         quadView.removeQuadrilateral()
         captureSessionManager?.start()
         UIApplication.shared.isIdleTimerDisabled = true
-        navigationController?.setNavigationBarHidden(true, animated: false)
+        
+        let statusBarHeight = UIApplication.shared.statusBarFrame.size.height
+        let visualEffectRect = self.navigationController?.navigationBar.bounds.insetBy(dx: 0, dy: -(statusBarHeight)).offsetBy(dx: 0, dy: -statusBarHeight)
+        
+        visualEffectView.frame = visualEffectRect ?? CGRect.zero
+
+        navigationController?.navigationBar.isTranslucent = true
```

Comment:
We set `navigationController?.navigationBar.isTranslucent` to 2 different values (`true`, and `false`), in 3 different places: `viewDidLoad`, `viewWillAppear` and `viewDidLayoutSubviews`. This should only be set once and probably in `viewDidLoad`.
This is probably true for the other properties as well

Reply:
It's only being set in the initial setup (`setupNavigationBar`) and in `viewWillAppear`/`viewWillDisappear`, because otherwise the property would stay the same after navigating to a different screen such as the edit screen.

Reply:
Do we need to set it in `setupNavigationBar` (`viewDidLoad`) since it's called it `viewWillAppear`?

Reply:
Fixed!